File: magma-platform/magma-platform-bungee/src/main/java/de/jalumu/magma/platform/bungee/command/TestCommand.java
package de.jalumu.magma.platform.bungee.command;

import de.jalumu.magma.command.MagmaCommand;
import de.jalumu.magma.player.MagmaPlayer;
import de.jalumu.magma.text.notification.Notification;

public class TestCommand {

    @MagmaCommand(command = "Test", description = "", usage = "test", permission = "test")
    public void test(MagmaPlayer player){
        Notification.info("Test");
    }

}

File: magma-platform/magma-platform-bungee/src/main/kotlin/de/jalumu/magma/platform/bungee/command/TestCommand.kt
package de.jalumu.magma.platform.bungee.command

import de.jalumu.magma.command.MagmaCommand
import de.jalumu.magma.player.MagmaPlayer
import de.jalumu.magma.text.notification.Notification

class TestCommand {
    @MagmaCommand(command = ["Test"], description = "", usage = "test", permission = "test")
    fun test(player: MagmaPlayer?) {
        Notification.info("Test")
    }
}
